DEP deviance in favor of log_loss for GradientBoostingClassifier (#23036)
scikit-learn/scikit-learn: doc/whats_new/v1.1.rst

    :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
    where 123456 is the *pull request* number, not the issue number.

- |API| The option for using the log loss, aka binomial or multinomial deviance, via

>   the ``loss` parameters was made more consistent. The preferred way is by

Reviewer: ```suggestion
  the `loss` parameters was made more consistent. The preferred way is by
```

>   setting the value to `"log_loss"`. Old option names are still valid,
>   produce the same models, but are deprecated and will be removed in version

Reviewer: ```suggestion
  setting the value to `"log_loss"`. Old option names are still valid and
  produce the same models, but are deprecated and will be removed in version
```

  1.3.

>   - For :class:`ensemble.GradientBoostingClassifier`, `loss="deviance"` is deprecated,
>     use `"log_loss"` instead which is now the default.

Reviewer 2: ```suggestion
- For :class:`ensemble.GradientBoostingClassifier`, The `loss` parameter name "deviance"
  is deprecated in favor of the new name "log_loss", which is now the default.
```
